composer.json

```diff
@@ -10,7 +10,8 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/symfony": ">=2.7",
+        "symfony/console": "~2.7|~3.0|~4.0",
+        "symfony/framework-bundle": "~2.7|~3.0|~4.0",
```

Comment:
Seems great for now, since `framework-bundle` is packed with `http-foundation` and `dependency-injection`.
Keep me posted on your testing with this set of dependecies, I'll try and test it myself too.

Still need to think about buzz though.

Reply:
I'm going to look at bundle testing over the next few days

Reply:
I haven't had time to test it, I wasn't able to find any time over the past few days to do so (busy travelling and talking)

Reply:
No problem, I totally understand.
I'll try and get some time over the week-end if possible!